bucket/sarasa-gothic.json

```diff
@@ -0,0 +1,31 @@
+{
+    "homepage": "https://github.com/be5invis/Sarasa-Gothic",
+    "description": "A CJK programming font based on Iosevka and Source Han Sans",
+    "version": "0.21.0",
+    "license": "OFL-1.1",
+    "url": "https://github.com/be5invis/Sarasa-Gothic/releases/download/v0.21.0/sarasa-gothic-ttf-0.21.0.7z",
+    "hash": "e127bcc3883ff6407e9a8cbf115cb736f843dd2e6b5cd986134bfa1ec74ab67c",
+    "installer": {
+        "script": [
+            "New-Item -Path 'HKCU:\\SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\Fonts\\Sarasa Gothic' -Force -ErrorAction SilentlyContinue | Out-Null",
```

Comment:
install for current user only, so these manifest/fonts can't install for other users via global installation.